bridge/opencensus/examples/bridge/go.mod

```
@@ -0,0 +1,18 @@
+module go.opentelemetry.io/otel/bridge/opencensus/examples/simple
```

This doesn't match the directory structure, should it be this:

```suggestion
module go.opentelemetry.io/otel/bridge/opencensus/examples/bridge
```